install/firmware.csv

```diff
@@ -324,6 +324,7 @@ CSI,EQR120,46540,m001.03.08,2022-03-31T22:39:00Z,9999-01-01T00:00:00Z,
 CSI,EQR120,46541,m001.03.08,2022-06-23T01:55:00Z,9999-01-01T00:00:00Z,
 CSI,EQR120,46542,m001.03.08,2022-06-23T01:55:00Z,9999-01-01T00:00:00Z,
 CSI,EQR120,46543,m001.03.08,2022-06-23T01:55:00Z,9999-01-01T00:00:00Z,
+CSI,EQR120,46595,m001.03.08,2023-01-12T23:45:00Z,9999-01-01T00:00:00Z,
```

Comment:
Firmware is ` 2.2.0`, sorry will update tickets

Reply:
thanks! ... will do